Mathlib/Combinatorics/SimpleGraph/Path.lean

```diff
@@ -1037,6 +1038,17 @@ lemma top_supp_eq_univ (c : ConnectedComponent (⊤ : SimpleGraph V)) :
 
 end ConnectedComponent
 
+lemma disjoint_deleteVerts_connectedComponentSupp {s : Set V}
+    {K : ((⊤ : Subgraph G).deleteVerts s).coe.ConnectedComponent} :
+    Disjoint s (Subtype.val '' K.supp) := by
```

Comment:
Similarly for these two - the type of `K` looks overly specific, and without seeing what you use it for it's difficult to justify the value of this one

Reply:
Done:)